include/swift/AST/DiagnosticsSema.def

```diff
@@ -4631,6 +4631,9 @@ ERROR(distributed_actor_func_param_not_codable,none,
 ERROR(distributed_actor_target_result_not_codable,none,
       "result type %0 of %1 %2 does not conform to serialization requirement '%3'",
       (Type, DescriptiveDeclKind, Identifier, StringRef))
+ERROR(distributed_actor_system_serialization_req_must_be_protocol,none,
+      "'SerializationRequirement' type witness %0 must be a protocol type",
```

Comment:
Maybe - `'SerializationRequirement' requires [a] value of [a] protocol type` or something like this? WDYT, @hborla and @xwu?

Reply:
I definitely think the term "type witness" should not be used in diagnostics. The requirement also isn't satisfied by a value, but rather a protocol conformance requirement. What about `'SerializationRequirement' must be satisfied by a protocol`? Perhaps with a note that says `'SerializationRequirement' is used as a protocol conformance requirement`

Reply:
`... must be satisfied by protocol or protocol composition` then maybe? This is going to be diagnosed in `typealias SerializationRequirement = any ...` position.

Reply:
Thanks! I'll reword somehow like this hmm